### PR TITLE
Turn Sterling off for all tests

### PR DESCRIPTION
**Pull request by sidprasad**

### CODE REVIEW

**Comment by bennn** [comment]
Oops, revert the `main.rkt` change. I already merged #164

File: forge/tests/forge/library/seq.frg [hunk 1]
```diff
@@ -1,4 +1,10 @@
-#lang forge/bsl
+#lang forge
+
+option run_sterling off
+/bsl
```
Comment (bennn) [inline forge/tests/forge/library/seq.frg:4]: Uh oh

**Review by bennn — CHANGES_REQUESTED** [review]

File: forge/tests/forge/other/open.frg [hunk 2]
```diff
@@ -1,4 +1,7 @@
 #lang forge
+
+option run_sterling off
+
 open "other-file.frg"
```
Comment (bennn) [inline forge/tests/forge/other/open.frg:5]: Oh, `open` has to be the first thing in the file.

File: forge/tests/forge/library/seq.frg (hunk 1 above)
Comment (sidprasad) [reply forge/tests/forge/library/seq.frg]: Oops, bad regex on my end when making the change

**Pull request merged**